docs/Server.md

```diff
@@ -82,6 +83,17 @@ is in use. Also, when `serverFactory` option is specified, this option is ignore
 
 + Default: `5000` (5 seconds)
 
+<a name="factory-max-requests-per-socket"></a>
+### `maxRequestsPerSocket`
+
+Defines the maximum number of requests socket can handle before closing keep alive connection. See documentation for
+[`server.maxRequestsPerSocket` property](https://nodejs.org/dist/latest-v16.x/docs/api/http.html#http_server_maxrequestspersocket)
+to understand the effect of this option. This option only applies when HTTP/1
+is in use. Also, when `serverFactory` option is specified, this option is ignored.
+>  Only node version greater or equal to 16.10.0 support this option.
+
++ Default: `null` (no limit)
```

Comment:
I would prefer to have `0` instead of `null` to indicate no limit.

Reply:
I need to test against the `http` server to see if `0` work as `null`.